Resources/Prototypes/Entities/Objects/Misc/extinguisher_spray.yml

```diff
@@ -0,0 +1,31 @@
+﻿- type: entity
+  name: Extinguisher Spray
+  parent: BaseItem
```

Comment:
I don't think you need to inherit BaseItem? This is only the effect right? Should someone pick this up?

Just tested ingame: you can pick it up...